cmake/configs/default.cmake

```diff
@@ -103,7 +103,7 @@ hunter_default_version(PhysUnits VERSION 1.1.0-p0)
 hunter_default_version(PocoCpp VERSION 1.7.9-p1)
 hunter_default_version(PostgreSQL VERSION 10.0.0)
 hunter_default_version(Protobuf VERSION 3.5.2-p0)
-
+hunter_default_version(QTPropertyEditor VERSION 2.1.3-p0)
```

Comment:
It's `Qt`, not `QT`: https://www.qt.io/

Reply:
Do you suggest replacing QT to Qt everywhere?